service/src/test/java/dk/dbc/rr/oai/OaiBeanIT.java

```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright (C) 2019 DBC A/S (http://dbc.dk/)
  *
- * This is part of rr-oai-service
+ * This is match of rr-oai-service
```

Comment:
?

Reply:
annnnnnoying IDE, thinks words in comments should be changed too.